ocaml/xapi/message_forwarding.ml

```diff
@@ -4473,5 +4473,9 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
           | _ -> find_first_live rest
       in
       find_first_live other_hosts
+
+    let resync_host ~__context ~host =
+      info "Cluster_host.resync_host host: %s" (Ref.string_of (Helpers.get_localhost ~__context));
+      Xapi_cluster_host_helpers.resync_host ~__context ~host
```

Comment:
I think that pool_resync should do the forwarding using `do_op_on`, and likely move the implementation of pool_resync from xapi_cluster.ml here, since it is all about forwarding API calls.